transformer/transformer.go

```diff
@@ -202,20 +220,43 @@ func extractPreloads(dom *amphtml.DOM) []*rpb.Metadata_Preload {
 	return preloads
 }
 
-// setBaseURL derives the absolute base URL, and sets it on c.BaseURL. The value
-// is derived using the <base> href in the DOM, if it exists. If the href is
-// relative, it is parsed in the context of the document URL.
-// This must run after DocumentURL is set on the context.
-func setBaseURL(c *transformers.Context) {
-	if n, ok := htmlnode.FindNode(c.DOM.HeadNode, atom.Base); ok {
-		if v, ok := htmlnode.GetAttributeVal(n, "", "href"); ok {
-			if u, err := c.DocumentURL.Parse(v); err == nil {
-				c.BaseURL = u
-				return
+// defaultMaxAgeSeconds is the max-age to apply when there is an inline
+// amp-script without an explicit max-age. This is 1 day, to parallel the
+// security precautions put in place around service workers:
+// https://dev.chromium.org/Home/chromium-security/security-faq/service-worker-security-faq#TOC-Do-Service-Workers-live-forever-
+const defaultMaxAgeSeconds int32 = 86400  // number of seconds in a day
```

Comment:
optional. defaultInlineScriptMaxAgeSeconds maybe? It's verbose but only used in one place. Just seems a little generic to me.